src/main/java/org/jenkins_ci/plugins/flexible_publish/FlexiblePublisher.java

```diff
@@ -112,10 +112,11 @@ public boolean prebuild(final AbstractBuild<?, ?> build, final BuildListener lis
     @Override
     public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher, final BuildListener listener)
                                                                                                 throws InterruptedException, IOException {
+        boolean wholeResult = true;
         for (ConditionalPublisher publisher : publishers)
             if (!publisher.perform(build, launcher, listener))
-                setResult(build, Result.FAILURE);
-        return true;
+                wholeResult = false;
+        return wholeResult;
```

Comment:
I'm not sure I should notify an error to Jenkins when any of publishers fail.
Anyway, this never affects the behavior of Jenkins.